cli/core/hack/rpm/tanzu-cli.spec

```diff
@@ -0,0 +1,46 @@
+Name:       tanzu-cli
+Version:    %{cli_version}
+Release:    0
+License:    Apache 2.0
+URL:        https://github.com/vmware-tanzu/tanzu-cli/
+Vendor:     VMware
+Summary:    The Tanzu CLI
+
+%ifarch amd64
+%define arch amd64
+%endif
+
+%ifarch aarch64
+# TODO For now, we use the amd64 build for arm64
+%define arch amd64
+%endif
+
+%undefine _disable_source_fetch
+Source0:    https://github.com/vmware-tanzu/tanzu-framework/releases/download/v%{version}/tanzu-cli-linux-%{arch}.tar.gz
```

Comment:
we plan to produce the RPMs as post-release action ?

Reply:
Because we need the Tanzu CLI binary to be VMware SRP-compliant, we need to download that asset from Github.  Therefore yes, building these packages will need to be done once a release is complete with all assets.

Reply:
ok - this means we dont intend to ship RPMs as official/product artifacts, but only community/upstream supported ?

Reply:
I guess this will depend on if products want to include such packages.  But the plan at the moment, is to have the packages published through this upstream repo